DEVELOPMENT.md

```diff
@@ -58,11 +58,13 @@ We adhere to the [general Sourcegraph principles for testing](https://docs.sourc
 
 1.  If this is a non-patch release, update the changelog. Add a new section `## $MAJOR.MINOR` to [`CHANGELOG.md`](https://github.com/sourcegraph/src-cli/blob/main/CHANGELOG.md#unreleased) immediately under `## Unreleased changes`. Add new empty `Added`, `Changed`, `Fixed`, and `Removed` sections under `## Unreleased changes`.
 2.  Find the latest version (either via the releases tab on GitHub or via git tags) to determine which version you are releasing.
-3.  `VERSION=9.9.9 ./release.sh` (replace `9.9.9` with the version you are releasing)
-4.  GitHub will automatically perform the release via the [goreleaser action](https://github.com/sourcegraph/src-cli/actions?query=workflow%3AGoreleaser). Once it has finished, **you need to confirm**:
+3.  Update the `"version"` field in the file `npm-distribution/package.json` to match the version you are releasing.
```

Comment:
Can the release script automate that?

Reply:
Is it OK if the release step runs `git push origin main`? I already updated the release script to validate that the version in package.json matches but didn't feel comfortable pushing a commit to main without review/test-plan.

Reply:
Given that we need to merge the changelog entries anwyays, i think it's OK to require a PR before running `VERSION=X ./release.sh`

Reply:
I'm happy to follow up on this after merge. Gonna try to cut a new release to see it go live on npm